src/index.js

```diff
@@ -11,7 +11,8 @@ export {
 } from './keys'
 export {
   nextYear, nextMonth, nextHour, makeUUID4, updateQueryStringParameter,
-  isLaterVersion, isSameOriginAbsoluteUrl, hexStringToECPair, ecPairToHexString
+  isLaterVersion, isSameOriginAbsoluteUrl, hexStringToECPair, ecPairToHexString,
+  ecPairToAddress
```

Comment:
`Utils.ecPairToAddress` replaces `ecKeyPair.getAddress()` as it was dropped from the interface of `ECPair`